shield-funds/utxo-1631177.json

```diff
@@ -0,0 +1,7 @@
+{
+    "txid": "3md9M0OOpPBsF02Rp2b7CJZMpv093bjLuSCIG1RPioU=",
+    "script": "dqkU1mkF+eETNMCYyJs0OZcygn0KDi+IrA==",
+    "valueZat": "10000",
+    "height": "1631177",
+    "address": "t1dRJRY7GmyeykJnMH38mdQoaZtFhn1QmGz"
+}
```

Comment:
No newline at the end of the file, probably does no harm but our convention is to have all text files end in a newline.